packages/components/src/base-control/styles/base-control-styles.ts

```diff
@@ -33,19 +33,21 @@ export const StyledField = styled.div`
 `;
 
 const labelStyles = css`
+	${ baseLabelTypography };
+
 	display: inline-block;
 	margin-bottom: ${ space( 2 ) };
-`;
-
-export const StyledLabel = styled.label`
-	${ labelStyles }
 	/**
 	 * Removes Chrome/Safari/Firefox user agent stylesheet padding from
 	 * StyledLabel when it is rendered as a legend.
 	 */
 	padding: 0;
```

Comment:
This `legend` reset has been moved from being included only in `StyledLabel`, to being shared between both `StyledLabel` and `StyledVisualLabel `. This is because `StyledVisualLabel` (a.k.a. `BaseControl.VisualLabel`) is also likely to be used `as="legend"`.